steely/plugins/linden.py

```diff
@@ -130,6 +132,43 @@ def create_user(id, first_name):
     USERDB.insert(data)
     return data
 
+def handle_gex_sell_cards(user_id, ticker, profit):
+    REED_ID = '100018746416184'
```

Comment:
can this not be hardcoded for other instances of steely that aren't reed?
i think it's just bot.uid

Reply:
ah that's so much better, will change